frontend/src/app/offers/offers.service.ts

```diff
@@ -1,6 +1,7 @@
 import { Injectable } from '@angular/core';
```

Comment:
can we make some cleanup here, 3rd party imports first, 1st party imports second?